### ISSUE TITLE
Add aria-label to ion-input

### ISSUE THREAD

**Issue by Joosshhhh** [ionic-team/ionic#21569]
**Ionic version:**

[x] **4.x**

**I'm submitting a ...** 

[ ] bug report
[ x] feature request

**Current behavior:**

Currently there's no way to add an aria-label to the native input element ion-input wraps, this is failing accessibility test for us

**Expected behavior:**

Add aria label as a proxy to the native input element just like placeholder is

**Steps to reproduce:**

Create an ion-input and try assigning an aria-label then run the accessibility testing tool **axe** (can use google chrome extension). You will see this error **Form elements must have labels**
and since there's no way to add an aria-label to the the native input element that ion-input wraps, this issue only can be resolved by adding it into the framework.

**Related code:**

Below you can see how I added an aria-label to the ion-input but it doesn't add it to the native input element which is causing accessibility issues.

<img width="399" alt="Screen Shot 2020-06-17 at 2 10 24 PM" src="https://user-images.githubusercontent.com/16000022/84934158-f0d77a80-b0a4-11ea-8780-1b6295f39f64.png">

**Other information:**

**Ionic info:** 

```
Ionic:

   Ionic CLI                     : 6.10.0 (/Users/joshuamc/.nvm/versions/node/v12.16.2/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 4.10.0
   @angular-devkit/build-angular : 0.901.7
   @angular-devkit/schematics    : 9.1.7
   @angular/cli                  : 9.1.7
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v12.16.2 (/Users/joshuamc/.nvm/versions/node/v12.16.2/bin/node)
   npm    : 6.14.5
   OS     : macOS Catalina

```

**Comment by liamdebeasi** [comment]
Thanks for the issue! This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Ionic, please create a new issue and ensure the template is fully filled out.

Thank you for using Ionic!
